packages/webpack/test/scaffold.js

```diff
@@ -65,7 +74,13 @@ function convertToMap(input, path = '') {
   return result
 }
 
-function runScript(code, globals = {}) {
+const defaultGlobals = () => ({
```

Comment:
why does it only contain `console`?

Reply:
it's useful to have console there by default. The testcases are otherwise not depending on anything else. If you need anything else, you pass your own globals.